packages/@o3r/design/schemas/design-token.schema.json

```diff
@@ -79,6 +79,14 @@
         "o3rMetadata": {
           "description": "Additional information to provide to the metadata if generated",
           "$ref": "#/definitions/otterExtensionMetadata"
+        },
+        "o3rUnit": {
```

Comment:
Could you explain somewhere (issue, documentation, PR comment) what kind of Figma limitations it addresses?

Reply:
Let me reply here:
It is several limitations such as the unsupported size unit (em, rem, wh, etc...), the ignorance of % or the issue regarding metrics it wrongly consider as px.

Part of these limitation are known and considered in Figma roadmap, part as due to the Design Token extractor tool.

Reply:
Thanks for the explanation :)
It could be nice if we could have it in the Readme so that our users may be aware of this feature

Reply:
Added in readme